csharp/codeql-extractor.yml

```diff
@@ -16,3 +16,10 @@ file_types:
     extensions:
       - .cs
 legacy_qltest_extraction: true
+options:
+  compression:
+    title: The type of compression.
```

Comment:
The type of TRAP compression.

Reply:
Copying the title used in the Ruby draft.